wallet/wallet-controller/src/lib.rs

```diff
@@ -224,16 +224,16 @@ impl<T: NodeInterface + Clone + Send + Sync + 'static, W: WalletEvents> Controll
         Ok(wallet)
     }
 
-    fn make_backup_wallet_file(file_path: impl AsRef<Path>) -> Result<(), ControllerError<T>> {
+    fn make_backup_wallet_file(file_path: impl AsRef<Path>, version: u32) -> WalletResult<()> {
         let backup_name = file_path
             .as_ref()
             .file_name()
             .map(|file_name| {
                 let mut file_name = file_name.to_os_string();
-                file_name.push("_backup");
+                file_name.push(format!("_backup_{version}"));
```

Comment:
Please add a "v" before the version. I know this sounds stupid, but it helps people understand what the number means. 2 compared to v2 makes a big difference.